### DISCUSSION TITLE
Battery Charge Limit settings below 60% are unaccessible to apply.

### DISCUSSION THREAD

**Discussion by TUF-DASH-F15-2021** [seerge/g-helper#2996] [General]
At first I was able to change the battery limit up to 40%, but now it won't go below 60% when adjusted. My laptop model is ASUS TUF Dash F15 2021 (FX516PC). I have a screen recording attached for better understanding. I have looked into similar issues (their issues are all about charging beyond the limit, but that wasn't the case for me) but none had the same issue as mine. Thank you in advance.

https://github.com/user-attachments/assets/980daecb-5d6b-4fd0-9a0a-f2dcd6e8e40f

**Comment by seerge** [comment]
@TUF-DASH-F15-2021 hello, your device simply doesn't support anything below 60% as was reported by lots of people here https://github.com/seerge/g-helper/discussions/2408

**Reply by TUF-DASH-F15-2021** [reply]
Oh, I see thank you. I also have another problem regarding the keyboard lighting mode. It is stuck in static mode (breathing and strobe does not work). I also have a video of my concern. 

https://github.com/user-attachments/assets/95542da3-37ce-4c02-9f7f-18496b43fae8

**Reply by seerge** [reply]
Most likely you need to disable [Dynamic Lighitng in windows](https://github.com/seerge/g-helper/wiki/FAQ#backlight-doesnt-change-color-or-stays-blue). Does mode selection work in AC ?

**Reply by TUF-DASH-F15-2021** [reply]
I turned off the dynamic lighting but It does not solve the issue. It works well in AC before I uninstalled it to use ghelper.

**Reply by seerge** [reply]
Did you use official AC uninstaller ? Cause there is a good chance AC components are still running and overriding settings

**Reply by TUF-DASH-F15-2021** [reply]
Yes, I used the official uninstaller from ASUS. I checked in extra and there is no ASUS services running

**Reply by seerge** [reply]
Then I don't know why exactly you can't change backlight modes. Check in task manager if you have anything from Asus still running and upload a log from `%appdata%\GHelper`

**Reply by TUF-DASH-F15-2021** [reply]
The only running Asus in task manager is "Asus PTP Service" and "ASUS Hotplug Controller". 
[log.txt](https://github.com/user-attachments/files/16673897/log.txt)

**Reply by seerge** [reply]
From the log it looks like backlight mode change is supported and accepted 
```
20/08/2024 5:47:37 pm: AuraMode: AuraBreathe
20/08/2024 5:47:37 pm: TUF RGB = B4-01-FF-FF-FF-EB : 0
20/08/2024 5:47:37 pm: TUF RGB = B4-01-FF-FF-FF-EB : OK <---- mode set
```

You can try to do hard reset and reinstall asus system control interface to see if it helps as explained in https://github.com/seerge/g-helper/wiki/Troubleshooting

If it still doesn't work - the only way to check what is different comparing to AC is installing AC again and intercepting all it's calls as I was asking / explaining [in this thread](https://github.com/seerge/g-helper/issues/2973#issuecomment-2294867113)

P.S. But that would be quite weird as you have a very old device, and backlight control there is 100% supported (as other people use same model). While people from https://github.com/seerge/g-helper/issues/2973 have very new (2024 models)

**Reply by TUF-DASH-F15-2021** [reply]
Okay, I will try to do all that and update you once I'm done. Thanks

**Reply by TUF-DASH-F15-2021** [reply]
Hello, the common troubleshooting did not work. I used the IRPMon and used AC, these are the logs that I captured.

[AC1.log](https://github.com/user-attachments/files/16686088/AC1.log)

**Reply by seerge** [reply]
@TUF-DASH-F15-2021 ok, thanks

I see what is different in your case comparing to other TUFs

Try this build 
[GHelper.zip](https://github.com/user-attachments/files/16688454/GHelper.zip)

**Reply by TUF-DASH-F15-2021** [reply]
Hello, the backlight modes are all working now. Thank you! If any other issues occurs I will inform you, once again thank you very much.
